### PR TITLE
add image to an existing location works now

### PR DESCRIPTION
**Pull request by ochanje210**
This PR is including the following features as well:

- myPage
- Facebook Authentication

### CODE REVIEW

File: shared/components/Create.js [hunk 1]
```diff
@@ -302,8 +320,31 @@ class Create extends Component {
...
+    const address =`${HTTP}${SERVER_ADDR}${ENDPOINT_IMAGE}`;
+    fetch(address, {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: data
+    })
+    .then((response) => response.json())
```
Comment (ovekyc) [inline shared/components/Create.js]: You should check whether response contain error or not.
If response contain error property, It was a fail post
Check out APIDoc
https://goober.herokuapp.com/docs/#api-Image-addAnImage

Comment (ochanje210) [inline shared/components/Create.js]: great point. but first we need to discuss what to do when it fails.

File: shared/components/Create.js [hunk 2]
```diff
@@ -302,8 +320,31 @@ class Create extends Component {
     })
   }
 
-  handleAddExistingLocation() {
-    //todo: implement adding photo to an existing location
+  handleAddExistingLocation(itemKey, userKey, caption, image) {
+    const data = JSON.stringify({
```
Comment (ochanje210) [inline shared/components/Create.js]: deleted! thanks!

File: shared/components/Create.js (hunk 1 above)
Comment (ochanje210) [inline shared/components/Create.js]: @ovekyc @yuda1124 @HewonJeong @festinalente91 
have any ideas?

Comment (ochanje210) [inline shared/components/Create.js]: added todo comment for this

File: shared/components/Create.js [hunk 3]
```diff
@@ -276,8 +260,8 @@ class Create extends Component {
   }
 
   getAddressData() {
-    const uri = API_GEODATA + '?latlng=' + this.props.currentLocation.latitude +","+
-      this.props.currentLocation.longitude + '&key='+ API_KEY;
+    const uri = API_GEODATA + '?latlng=' + this.props.currentLocation.latitude + ',' +
```
Comment (ovekyc) [inline shared/components/Create.js]: Please make url using functions in utils.js file

File: shared/components/Create.js [hunk 4]
```diff
@@ -299,11 +283,38 @@ class Create extends Component {
...
   }
 
-  handleAddExistingLocation() {
-    //todo: implement adding photo to an existing location
+  // todo:  should check whether response contain error or not.
+  // If response contain error property, It was a fail post
+  // Check out APIDoc
+  // https://goober.herokuapp.com/docs/#api-Image-addAnImage
+  handleAddExistingLocation(itemKey, userKey, caption, image) {
+    const data = JSON.stringify({
```
Comment (ovekyc) [inline shared/components/Create.js]: You dont have to yse JSON.stringify

Comment (ochanje210) [inline shared/components/Create.js]: I had a problem on sending request without JSON.stringify.

**Comment by ovekyc** [comment]
Please separate PR as small chunk. This is too large. hard to reviewwww

**Comment by ochanje210** [comment]
:D why don't you review more frequently

File: shared/components/Create.js (hunk 3 above)
Comment (ochanje210) [inline shared/components/Create.js]: followed!

**Comment by ovekyc** [comment]
LGTM

**Pull request merged**